aspnet/tutorials/web-api-help-pages-using-swagger.rst

```diff
@@ -40,7 +40,7 @@ You can add Swashbuckle with any of the following approaches:
 
 .. code-block:: javascript
 
-    "Swashbuckle": "6.0.0-*"
+    "Swashbuckle": "6.0.0-beta902"
 
```

Comment:
Next-gen is still beta and I expect there still could be one or two more breaking changes to come. So, I think it may be prudent to tie the docs to a specific beta version